src/hooks/activity.ts

```diff
@@ -297,3 +302,27 @@ export const useActivityAdmin = (activityId: string) => {
     refetchActivityAdmin: refetch,
   }
 }
+
+
+export const useCategroyCollection = (condition: hasura.GET_ACTIVITIES_CATEGORIESVariables['condition']) => {
+  const { loading, error, data } = useQuery<
+    hasura.GET_ACTIVITIES_CATEGORIES,
+    hasura.GET_ACTIVITIES_CATEGORIESVariables
+  >(
+    gql`
+      query GET_ACTIVITIES_CATEGORIES($condition: activity_bool_exp) {
+        category(where: {activity_categories: {
+          activity: $condition
```

Comment:
How about order by position ascending in graphQL here instead sort by position line 326 ?

```
query GET_ACTIVITIES_CATEGORIES(...) {
        category(where: {...}, 
order_by: [{position: asc}]
) {
          id
          ... 
       }
      }
```

Reply:
Thanks for the suggestions. :D